internal/cmd/run/cypress.go

```diff
@@ -111,14 +111,19 @@ func runCypress(cmd *cobra.Command, tc testcomposer.Client, rs resto.Client, as
 
 	rs.ArtifactConfig = p.Artifacts.Download
 
-	tracker := segment.New()
+	var tracker *segment.Tracker
+	if !gFlags.disableSegment {
+		tracker = segment.New()
+	}
```

Comment:
I think that's a bit unwieldy to do everywhere.

The easier solution would be to init the tracker regardless of the setting, but when `Collect()` is called, you simply do nothing.

Reply:
make sense 👍